### PR TITLE
Document tables, part I

### PR DESCRIPTION
**Pull request by achamayou**
Aiming for complete coverage of `public:ccf.gov.*`

#230

### CODE REVIEW

**Review by wintersteiger — APPROVED** [review]
LGTM. I guess there isn't an easy way to connect the docs to the code so that we get notified if a table is renamed/removed/added/etc?

**Comment by achamayou** [comment]
@wintersteiger not very easily, but a sanity check may be to scrape the public tables from a ledger post-test case and check they’re at least all mentioned on this page, if not the other way around.

File: doc/audit/builtin_maps.rst [hunk 1]
```diff
@@ -1,29 +1,264 @@
 Built-in Maps
 =============
 
-`public:ccf.gov.nodes.code_ids`
--------------------------------
+`public:ccf.gov.`
```
Comment (jumaffre) [inline doc/audit/builtin_maps.rst:4]: Nitpick^2: Why italic?

File: doc/audit/builtin_maps.rst [hunk 2]
```diff
@@ -1,29 +1,264 @@
...
+**Key** Member ID: SHA-256 digest of the member certificate, represented as a hex-encoded string.
+
+**Value** Member certificate, represented as a PEM-encoded string.
+
+`encryption_public_keys`
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Public encryption keys submitted by members to the network. The recovery share for each member is encrypted by the key they have submitted.
+
+**Key** Member ID: SHA-256 digest of the member certificate, represented as a hex-encoded string.
```
Comment (jumaffre) [inline doc/audit/builtin_maps.rst:21]: Here and elsewhere:
```suggestion
**Key** Member ID: SHA-256 digest of the DER-encoded member certificate, represented as a hex-encoded string.
```

**Review by jumaffre — APPROVED** [review]

File: doc/audit/builtin_maps.rst (hunk 1 above)
Comment (achamayou) [reply doc/audit/builtin_maps.rst:4]: I was actually going for monospace, but I markdowned. Monospace seems good for table names? What do we think?

File: doc/audit/builtin_maps.rst (hunk 2 above)
Comment (achamayou) [reply doc/audit/builtin_maps.rst:21]: Yes, absolutely, very well spotted, I will fix this everywhere.

File: doc/audit/builtin_maps.rst (hunk 1 above)
Comment (jumaffre) [reply doc/audit/builtin_maps.rst:4]: Monospace is great!

File: doc/audit/builtin_maps.rst (hunk 2 above)
Comment (jumaffre) [reply doc/audit/builtin_maps.rst:21]: It's also often referred to as the SHA-256 fingerprint of the certificate, which is more concise.

**Pull request merged**